frontend/packages/local-storage-operator-plugin/integration-tests/tests/disks-list-page.scenario.ts

```diff
@@ -0,0 +1,91 @@
+import { $, ExpectedConditions as until, browser, Key } from 'protractor';
+import { execSync } from 'child_process';
+import { appHost } from '@console/internal-integration-tests/protractor.conf';
+import { resourceRows, isLoaded } from '@console/internal-integration-tests/views/crud.view';
+import { clickNavLink } from '@console/internal-integration-tests/views/sidenav.view';
+import {
+  page,
+  diskList,
+  clickFilterDropdown,
+  checkDiskFilter,
+  clearDiskFilter,
+} from '../views/disks-list-page.views';
+import { LOCAL_STORAGE_NAMESPACE } from '../../src/constants/index';
+import {
+  LocalVolumeDiscoveryResultKind,
+  DiskMetadata,
+  DiskStates,
+} from '../../src/components/disks-list/types';
+
+// Pre-requirement: Local volume discovery to be present
+describe('Disk List is accessible from Nodes view', () => {
+  let discoveredDevices: DiskMetadata[];
+  beforeAll(async () => {
+    const lvdrJson = JSON.parse(
+      execSync(
+        `kubectl get localvolumediscoveryresults -n ${LOCAL_STORAGE_NAMESPACE} -o json`,
+      ).toString(),
+    );
+    const lvdr: LocalVolumeDiscoveryResultKind = lvdrJson.items[0];
+    const { nodeName } = lvdr.spec;
+    discoveredDevices = lvdr.status.discoveredDevices;
+    await browser.get(`${appHost}/`);
+    await clickNavLink(['Compute', 'Nodes']);
+    await isLoaded();
+    $(`a[data-test-id="${nodeName}"]`).click();
+  });
+  it('`Disks` tab is enabled and accessible', async () => {
+    await browser.wait(until.presenceOf(page.diskTab), 5000);
+    page.diskTab.click();
```

Comment:
```suggestion
    click(page.diskTab);
```
Rather than clicking a better test would be to expect an `elementToBeClickable`.